index.js

```diff
@@ -1,26 +1,24 @@
 'use strict';
-var arrify = require('arrify');
-var devices = require('./data.json').devices;
+const arrify = require('arrify');
+const devices = require('./data.json').devices;
 
-module.exports = function (items) {
+module.exports = items => {
 	items = arrify(items);
 
 	if (!items.length) {
 		return devices;
 	}
 
-	items = items.map(function (item) {
-		return item.split(' ').join('').toLowerCase();
-	});
+	items = items.map(item => item.split(' ').join('').toLowerCase());
 
-	var ret = [];
+	const ret = [];
 
-	items.forEach(function (item) {
-		devices.filter(function (device) {
-			return device.name.split(' ').join('').indexOf(item) !== -1;
-		}).forEach(function (device) {
-			ret.push(device);
-		});
+	items.forEach(item => {
+		devices
+			.filter(device => device.name.split(' ').join('').indexOf(item) !== -1)
```

Comment:
You mean [includes](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/String/includes)?